content/docs/concepts/security/index.md

```diff
@@ -570,6 +570,8 @@ authorization result, `ALLOW` or `DENY`.
 
 ### Authorization permissive mode
```

Comment:
In the concept doc, since permissive mode is an experimental feature, let's move it to the end of authorization section (before "using other authorization mechanisms")?

Reply:
updated.